main.go

```diff
@@ -183,6 +151,67 @@ func main() {
 		log.Tracef("Toolexec final command:    %q\n", proxyCmd.Args())
 		proxy.MustRunCommand(proxyCmd)
 
+		return
+	case "warmup":
+		tmp, err := os.MkdirTemp("", "orchestrion-warmup-*")
+		if err != nil {
+			fmt.Fprintf(os.Stderr, "Unable to create temporary directory: %v\n", err)
+			os.Exit(1)
+		}
+		defer os.RemoveAll(tmp)
+
+		log.Tracef("Initializing warm-up module in %q\n", tmp)
+		var stderr bytes.Buffer
+		cmd := exec.Command("go", "mod", "init", "orchestrion-warmup")
+		cmd.Dir = tmp
+		cmd.Stderr = &stderr
+		if err := cmd.Run(); err != nil {
+			fmt.Fprintf(os.Stderr, "Unable to initialize temporary module: %v\n", err)
+			if stderr.Len() > 0 {
+				fmt.Fprintf(os.Stderr, "Error output:\n%s\n", stderr.String())
```

Comment:
Can you show the full command that was ran somewhere in the output for people to run it manually ?